src/databricks/sql/thrift_backend.py

```diff
@@ -55,6 +55,7 @@
 
 TIMESTAMP_AS_STRING_CONFIG = "spark.thriftserver.arrowBasedRowSet.timestampAsString"
 DEFAULT_SOCKET_TIMEOUT = float(900)
+DEFAULT_STATEMENT_HEARTBEAT_INTERVAL = float(25)
```

Comment:
For me 25 seconds delay between GetOperationStatus requests looks a bit too much. I mean - if query execution finishes faster than this delay - client will still wait that 25 seconds. In Nodejs we poll for operation status with 100ms interval - which also may be too low, but looks way more reasonable

Reply:
100ms?  Do you actually see that in practice?  Today we wait 0, and the server holds up for up to 5 seconds.  Seems weird to care about subsecond latency on queries that we know take more than 5 seconds.